data/test/scenarios/wml_tests/FilterWML/special_id_active.cfg

```diff
@@ -0,0 +1,111 @@
+# This unit test reproduces setup of weapon special such as https://github.com/inferno8/wesnoth-Era_of_Magic/blob/adf227cba2e66602c4bb49dc203fa751865aa124/utils/beam.cfg, which is not supposed to work in rpg.
+# Focus of this test is whether [filter_attack]special_id_active work for [filter_self]lua_function controlled conditional weapon special
```

Comment:
Please put comments in the "API(s) being tested:" template format, and add the extras that that template suggests.

Line 1 is good and should stay, but line 2 would fit in the standard template.

Reply:
updated and named file to match test id